examples/src/main/java/com/mapbox/navigation/examples/activity/SimpleMapboxNavigationKt.kt

```diff
@@ -106,20 +119,32 @@ class SimpleMapboxNavigationKt : AppCompatActivity(), OnMapReadyCallback {
         }
     }
 
-    private val routeObserver = object : RouteObserver {
+    private val routesObserver = object : RoutesObserver {
         override fun onRoutesChanged(routes: List<DirectionsRoute>) {
-            navigationMapRoute?.addRoute(routes[0])
+            navigationMapRoute?.addRoutes(routes)
```

Comment:
Shouldn't the if check be before this line. I mean why would we want to `addRoutes()` if `routes` is empty

Reply:
The goal was to clear the route lines if the directions session is reporting empty. Looking at the UI lib implementation now, an empty list will simply be ignored. I'll play with `#updateRouteVisibilityTo` instead.